sybil/parsers/codeblock.py

```diff
@@ -32,11 +32,28 @@ class CodeBlockParser(object):
         in each of the examples found by this parser.
     """
 
+    _LANGUAGES = {
```

Comment:
```suggestion
    languages = {
```

Not a constant and doesn't need to be private :-)

Reply:
I added the `add_evaluator()` method specifically to keep this private. I didn't want users to start modifying the dict directly. They shouldn't care about how the dispatch is done.